src/server/index.js

```diff
@@ -5,7 +5,17 @@ const app = express();
 
 app.use(express.static('dist'));
 app.get('/heartbeat', (req, res) => {
-  res.json({status: 'ok'});
+  var response;
+  var MongoClient = require('mongodb').MongoClient;
+  var url = 'mongodb://localhost:27017/epam';
```

Comment:
you have to read the settings from the environment config, that is why we use the .env module. Other developers will may use other urls and ports as well as on the staging environment.